tests/vite-basic-compat/ember-cli-build.js

```diff
@@ -3,8 +3,18 @@
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 const { maybeEmbroider } = require('@embroider/test-setup');
 
-module.exports = function (defaults) {
+module.exports = async function (defaults) {
+  const { setConfig } = await import('@warp-drive/build-config');
+
   let app = new EmberApp(defaults, {});
 
+  /**
+   * TODO: find another way to set this config,
+   *       because the most modern of apps will not have a compat build
+   */
```

Comment:
once embroider/macros picks a new key that's all we need. `app` and `__dirname` here are just for the macros API that needs them.

Reply:
(probably should remove this comment though)